schema/url/data/pass/plugin-scatterplot.raml

```diff
@@ -34,6 +34,8 @@ types:
         type: integer
         format: int64
         required: false
+      correlationMethod:
```

Comment:
Maybe we talked about this already before but the name `SelfCorrelationMethod` is a bit odd here, since it doesn't have to be a self correlation that we're running. I know that `SelfCorrelationMethod` contians all the correlation methods, but that's unclear from the name.

Reply:
Also should this be optional?